src/panels/config/script/ha-script-editor.ts

```diff
@@ -110,164 +149,215 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
-                        rel="noreferrer"
-                      >
+                              ${this.hass.localize(
+                                "ui.panel.config.script.editor.link_available_actions"
+                              )}
+                            </a>
+                          </span>
+                          <ha-automation-action
+                            .actions=${this._config.sequence}
+                            @value-changed=${this._sequenceChanged}
+                            .hass=${this.hass}
+                          ></ha-automation-action>
+                        </ha-config-section>
+                      `
+                    : ""}
+                </div>
+              `
+            : this._mode === "yaml"
+            ? html`
+                <ha-config-section .isWide=${false}>
+                  ${!this.narrow
+                    ? html` <span slot="header">${this._config?.alias}</span> `
+                    : ``}
+                  <ha-card>
+                    <div class="card-content">
+                      <ha-yaml-editor
+                        .defaultValue=${this._preprocessYaml()}
+                        @value-changed=${this._yamlChanged}
+                      ></ha-yaml-editor>
+                      <mwc-button @click=${this._copyYaml}>
                         ${this.hass.localize(
-                          "ui.panel.config.script.editor.link_available_actions"
+                          "ui.panel.config.automation.editor.copy_to_clipboard"
                         )}
-                      </a>
-                    </span>
-                    <ha-automation-action
-                      .actions=${this._config.sequence}
-                      @value-changed=${this._sequenceChanged}
-                      .hass=${this.hass}
-                    ></ha-automation-action>
-                  </ha-config-section>
-                `
-              : ""}
-          </div>
+                      </mwc-button>
+                    </div>
+                    ${this.scriptEntityId
+                      ? html`
+                          <div
+                            class="card-actions layout horizontal justified center"
+                          >
+                            <span></span>
+                            <mwc-button
+                              @click=${this._runScript}
+                              title="${this.hass.localize(
+                                "ui.panel.config.script.picker.activate_script"
+                              )}"
+                              ?disabled=${this._dirty}
+                            >
+                              ${this.hass.localize("ui.card.script.execute")}
+                            </mwc-button>
+                          </div>
+                        `
+                      : ``}
+                  </ha-card>
+                  <ha-config-section> </ha-config-section
```

Comment:
?
```suggestion
```

Reply:
Actually I think part of this line is on the next line so my suggestion is bad but I don't think this is needed

Reply:
I was sure I had removed this already...